LPFilter/Source/PluginProcessor.cpp

```diff
@@ -18,16 +18,22 @@ LpfilterAudioProcessor::LpfilterAudioProcessor()
      : AudioProcessor (BusesProperties()
                      #if ! JucePlugin_IsMidiEffect
                       #if ! JucePlugin_IsSynth
+                       // Sets Input and Output bus to be stereo by default
                        .withInput  ("Input",  AudioChannelSet::stereo(), true)
                       #endif
                        .withOutput ("Output", AudioChannelSet::stereo(), true)
                      #endif
-                       ) // Sets Input and Output bus to be stereo by default
+                       ),
+        // Setup lpfJuce
+        lpfJuce(dsp::IIR::Coefficients<float>::makeFirstOrderLowPass(44100.0, 60.f))
```

Comment:
- [ ] Move this line outside `#ifndef JucePlugin_PreferredChannelConfigurations` statement
- [ ] The default value of frequency (60.0f) is a hard coded value in lines 28 & 34. You should create a temp variable, assign it the value 60.0f and use it in the above statements. In general , duplicated code makes code readability complex and might cause bugs.